examples/autoqasm/4_Native_programming.ipynb

```diff
@@ -46,12 +46,14 @@
      "output_type": "stream",
      "text": [
       "OPENQASM 3.0;\n",
+      "output bit[2] retval_;\n",
```

Comment:
I see `retval_` in the output cells for this notebook - does it need to be re-run with the latest changes?

Reply:
Ahh, I thought I'd ran them again, thank you for catching this